alkemio.yml

```
@@ -278,6 +278,9 @@ communications:
 ## storage ##
 # Alkemio uses multiple types of persistent storage, including SQL database and IPFS.
 storage:
+  graphql_upload:
+    # 20MB
+    max_file_size: 1048576
```

1048576 is 10, not 20
```suggestion
    max_file_size: 20971520
```